enginetest/load_queries.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package enginetest
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+const (
+	tableNameConst = "LOADTABLE"
+)
+
+var LoadDataScripts = []ScriptTest{
+	{
+		Name: "Super basic load data",
+		SetUpScript: []string{
+			fmt.Sprintf("create table %s(pk int primary key)", tableNameConst),
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    fmt.Sprintf("select * from %s", tableNameConst),
+				Expected: []sql.Row{{int8(1)}, {int8(2)}, {int8(3)}, {int8(4)}},
+			},
+		},
+	},
+	{
+		Name: "Load data with csv",
+		SetUpScript: []string{
+			fmt.Sprintf("create table %s(pk int primary key, c1 longtext)", tableNameConst),
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    fmt.Sprintf("select * from %s", tableNameConst),
+				Expected: []sql.Row{{int8(1), "hi"}, {int8(2), "hello"}},
+			},
+		},
+	},
+	{
+		Name: "Load data with csv with prefix.",
+		SetUpScript: []string{
+			fmt.Sprintf("create table %s(pk longtext primary key, c1 int)", tableNameConst),
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    fmt.Sprintf("select * from %s", tableNameConst),
+				Expected: []sql.Row{{"\"abc\"", int8(1)}, {"\"def\"", int8(2)}},
+			},
+		},
+	},
+}
+
+// Creates a directory of files
+func CreateDummyFiles(dir string) error {
```

Comment:
This is a confusing way to set up test data. The test data should be declared alongside the test scenario.

My suggestion is to just check files into source control in a testdata directory, and then issue actual `LOAD DATA` commands for the various files in the setup portion of the scripts above.